lib/src/parse/stylesheet.dart

```diff
@@ -1064,7 +1074,7 @@ abstract class StylesheetParser extends Parser {
   ImportRule _importRule(LineScannerState start) {
     var imports = <Import>[];
     do {
-      whitespace();
+      whitespace(allowNewlines: false);
```

Comment:
Eventually `@import` will be parsed as a plain-CSS at-rule, so we should be more conservative about allowing newlines to avoid a forwards-incompatibility issue.